directory_forms_api_client/version.py

```diff
@@ -1 +1 @@
-__version__ = '4.1.0'
+__version__ = '5.0.0'
```

Comment:
please also move the version info exclusively to setup.py

see [here](https://github.com/uktrade/directory-cms-client/pull/44/files#diff-2eeaed663bd0d25b7e608891384b7298) as example